modules/understanding-upgrade-channels.adoc

```diff
@@ -1,11 +1,12 @@
 // Module included in the following assemblies:
 //
-// updating/updating-cluster.adoc
-// updating/updating-cluster-cli.adoc
-// updating/updating-cluster-rhel-compute.adoc
-// updating/updating-disconnected-cluster.adoc
+// * updating/updating-cluster.adoc
+// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster-rhel-compute.adoc
+// * updating/updating-disconnected-cluster.adoc
 
-[id="understanding_upgrade_channels_{context}"]
+[id="understanding-upgrade-channels_{context}"]
```

Comment:
Not critical, just wondering whether we try to avoid changing anchor links in a version after it's been released.

Reply:
It's an error that I regret not catching during peer review. There was a lot of drama in the mod docs repo over how much using malformed anchor IDS can break automation. There are also no other links to this content in the collection.